bonsai/Data/Entity/Budget.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Budget.swift
+//  bonsai
+//
+//  Created by antuan.khoanh on 19/06/2022.
+//
+
+import Foundation
+import CoreData
+
+@objc(Budget)
+public class Budget: NSManagedObject, Identifiable {
+    @NSManaged public var name: String
+    @NSManaged public var totalAmount: NSDecimalNumber
+    @NSManaged public var currentAmount: NSDecimalNumber
```

Comment:
Not sure that we need to store current amount
Because amount depends of transactions it might be better to calculate it instead

Reply:
+updated